src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.test.ts

```diff
@@ -0,0 +1,54 @@
+import { describe, expect, it } from 'vitest'
+
+import { addAggregatorAndDestinationToMemo } from './addAggregatorAndDestinationToMemo'
+import { MEMO_PART_DELIMITER } from './constants'
+
+const RECEIVE_ADDRESS = '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
+const AGGREGATOR_ADDRESS = '0xd31f7e39afECEc4855fecc51b693F9A0Cec49fd2'
+const FINAL_ASSET_ADDRESS = '0x8a65ac0E23F31979db06Ec62Af62b132a6dF4741'
+
+describe('addAggregatorAndDestinationToMemo', () => {
```

Comment:
Not for your PR but note to self - https://github.com/shapeshift/web/pull/6755/commits/0a937ac73eaad687d8e8924dc27e1a6a79c2d703 is coming in to theaters around you soon, if your PR goes in first (which is very likely), then we should probably refactor this in #6755 to be composable, support aggregator and destination as part of the memo, and allow mutating it